packages/oak-addon-richtext-field/lib/core/Editor/html.js

```diff
@@ -122,7 +134,7 @@ export const deserializeNode = el => {
   }
 
   const children = Array.from(el.childNodes)
-    .map(deserializeNode);
+    .map(deserializeNode).flat();
```

Comment:
yea I don't think `.flat` is dangerous per se here as it won't flatten a non-empty array, but it would have been great to understand why it is necessary.